creator-cards/_creatorTemplate.json

```diff
@@ -13,23 +13,19 @@
     "sessionize": "",
     "other": ""
   },
-  "channels": [
+  "channels": [ // You must have published at least three post (long form content) since 2022 on any of your channels. About .NET or SPA frontend technologies
     {
-      "name": "Site | Twitch | Youtube | etc",
-      "url": "https://ilovedotnet.org"
+      "name": "Blog|Site|Twitch|Youtube|Book|etc",
+      "url": "https://www.wearedotnet.io/"
     }
   ],
-  "feeds": [
```

Comment:
lets please stick to the feeds array and just use the type. 
currently the type isn't used at all. We could just change that to these types:

dotnet, 
personal,


any dotnet feed must follow the rules, personal is only visible on profile page but can contain other topics as well